src/org/labkey/test/tests/AdminConsoleTest.java

```diff
@@ -157,10 +160,11 @@ public void testAppAdminRole()
         assertElementPresent("expect to return to admin console", siteAdminLoc, 1);
 
         //authentication
-        goToAdminConsole().clickAuthentication();
-        assertElementNotPresent("expect 'enable' links to be disabled for appAdmin", Locator.linkWithText("enable"));
-        assertElementNotPresent("expect 'configure' links to be disabled for appAdmin", Locator.linkWithText("configure"));
-        clickButton("Done");
+        LoginConfigurePage configurePage = goToAdminConsole().clickAuthentication();
+        List<LoginConfigRow> configRows = configurePage.getPrimaryConfigurations();
+        assertFalse("expect 'edit' links not to be available for auth configs", configRows.stream().anyMatch(a-> a.canEdit()));
```

Comment:
We should perform the inverse of this check as a site admin to make sure it doesn't just always return false. Either because there aren't any configs to edit or because the Locator in `canEdit` becomes invalid.

Reply:
I've pulled the locator out into an elementCache member; now canEdit() references the same locator as the element used in the cache every time we edit a configuration